test_runner/fixtures/neon_fixtures.py

```diff
@@ -286,19 +286,15 @@ def port_distributor(worker_base_port: int) -> PortDistributor:
     return PortDistributor(base_port=worker_base_port, port_number=WORKER_PORT_NUM)
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="function")
 def default_broker(
-    request: FixtureRequest,
     port_distributor: PortDistributor,
-    top_output_dir: Path,
+    test_output_dir: Path,
```

Comment:
And now we can delete `allure_attach_from_dir(Path(broker_logfile))` at the end of the method